### PR TITLE
Fix import path in proto files

### PR DESCRIPTION
**Pull request by zhejiangxiaomai**

### CODE REVIEW

**Comment by mbasmanova** [comment]
@zhejiangxiaomai Let's add compiling of this code to CircleCI job. Otherwise, we don't get a signal from CI whether the code compiles or not. Maybe, linux-build.

@kgpai Krishna, I believe this change breaks buck build. Is this something you could help look into?

**Comment by zhejiangxiaomai** [comment]
> @zhejiangxiaomai Let's add compiling of this code to CircleCI job. Otherwise, we don't get a signal from CI whether the code compiles or not. Maybe, linux-build.
> 
> @kgpai Krishna, I believe this change breaks buck build. Is this something you could help look into?

Yes,  I think so. I will add compiling task to CircleCI job.

File: velox/substrait/proto/substrait/algebra.proto [hunk 1]
```diff
@@ -2,8 +2,8 @@ syntax = "proto3";
 
 package substrait;
 
-import "velox/substrait/proto/substrait/type.proto";
```
[the remark this answers is not in the document]
Comment (mbasmanova) [reply velox/substrait/proto/substrait/algebra.proto:5]: I believe this code is not running in CircleCI today. We need to add it.

[the remark this answers is not in the document]
Comment (zhejiangxiaomai) [reply velox/substrait/proto/substrait/algebra.proto:5]: I added compile task in CI. and test it on my own machine. It is useful.

[the remark this answers is not in the document]
Comment (zhejiangxiaomai) [reply velox/substrait/proto/substrait/algebra.proto:5]: > I believe this code is not running in CircleCI today. We need to add it.

Hi @mbasmanova , all CircleCI  are successful, you can merge this PR.

**Comment by zhejiangxiaomai** [comment]
@kgpai some Facebook Internal -  Build& test failed.  can you show me the error information.

**Comment by kgpai** [comment]
@zhejiangxiaomai looks like the new paths dont work with metas internal systems. A lot of file not found errors.

**Comment by zhejiangxiaomai** [comment]
> @zhejiangxiaomai looks like the new paths dont work with metas internal systems. A lot of file not found errors.

Can you give me some suggestions to solve this problem? because this relative path work well in github circleci.

**Comment by kgpai** [comment]
@zhejiangxiaomai Would it be possible to do something CircleCI side. I will work in exploring that possibility.

**Comment by zhejiangxiaomai** [comment]
> @zhejiangxiaomai Would it be possible to do something CircleCI side. I will work in exploring that possibility.

Thank you.

**Comment by zhejiangxiaomai** [comment]
Hi, @kgpai  did you fix this problem about internal test?

**Comment by kgpai** [comment]
Hi @zhejiangxiaomai  I did not unfortunately get the time for this yet. I plan to work on this early next week .

**Comment by kgpai** [comment]
Update:  @zhejiangxiaomai  I am starting work on this now.

**Comment by kgpai** [comment]
@zhejiangxiaomai I opened #2000 which fixes this without modifying the protofiles. I just made a change to the cmake files and used your corresponding config.yaml changes. Please have a look and comment.

**Comment by zhejiangxiaomai** [comment]
> @zhejiangxiaomai I opened #2000 which fixes this without modifying the protofiles. I just made a change to the cmake files and used your corresponding config.yaml changes. Please have a look and comment.

Ok, thank you.

**Comment by kgpai** [comment]
@zhejiangxiaomai Merged #2000 . I am going to close this PR . Please reopen if required.

**Pull request closed without merging**